SpatialGDK/Documentation/content/troubleshooting.md

```diff
@@ -6,9 +6,9 @@ There could be a few different reasons for this. The list below provides some of
 
 1. It's easy to forget to generate the schema for your replicated Actor. Make sure you run the Schema Generator before launching your project.
 
-2. As per Unreal Engine’s [replication documentation](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors), your Actor needs to be created on the server-worker before it can replicate to the client-workers.
+2. As per Unreal Engine’s [replication documentation](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors), your Actor needs to be created on the server-worker instance before it can replicate to the client-worker instances.
```

Comment:
```suggestion
2. As described in Unreal's [replication documentation](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors), your game needs to create an Actor on server-worker instances before it can replicate to client-worker instances.
```
Had to get rif of "per".